python-package/lightgbm/sklearn.py

```diff
@@ -333,6 +334,10 @@ def fit(self, X, y,
         """
         evals_result = {}
         params = self.get_params()
+        # sklearn interface has another naming convention
+        params['seed'] = params.pop('seed', params.pop('random_state'))
```

Comment:
OK, I'll add warnings.
Do we need description of deprecated names into `__init__` docstring or warnings are enough?